languages.toml

```diff
@@ -4371,3 +4371,16 @@ injection-regex = "rust-format-args"
 [[grammar]]
 name = "rust-format-args"
 source = { git = "https://github.com/nik-rev/tree-sitter-rustfmt", rev = "2ca0bdd763d0c9dbb1d0bd14aea7544cbe81309c" }
+
+[[language]]
+name = "alloy"
+scope = "source.alloy"
+injection-regex = "alloy"
+file-types = ["alloy"]
+comment-token = "//"
+block-comment-tokens = { start = "/*", end = "*/" }
+indent = { tab-width = 2, unit = "  " }
+
+[[grammar]]
+name = "alloy"
+source = { git = "https://github.com/mattsre/tree-sitter-alloy", rev = "cfe318ee01ff68c43d1776733134d293975f664d" }
```

Comment:
Could you add a license to tree-sitter-alloy and update the commit here that once it's included? That will help packagers downstream.

Reply:
Ah good point I hadn't noticed that

\cc @nik-rev could you add a license to `tree-sitter-rust-format-args`?